.dynamic/verified-hashes.json

```diff
@@ -28,5 +28,15 @@
         "url": "https://github.com/small-tech/should-i-pipe-it/pull/3"
       }
     ]
+  },
+  "a7eb95476659f5482345376c09cf7bbcc491490d2aca6751de554dcec229cfe9fc2167ea1ec50021560694360abe3a096340301e28540db09ac3615f61c25157": {
+    "url": "https://sh.rustup.rs",
+    "verifiers": [
+      {
+        "name": "rugk",
+        "isAuthorOfScript": false,
+        "url": "https://github.com/small-tech/should-i-pipe-it/pull/4"
```

Comment:
PS. This link was incorrect, I just updated it to the correct one. GitHub shares the index between issues and pull requests so it’s good to check exactly what the index you’re getting on the pull request is. Barring a race condition, it will be one higher than `max(currentIssueIndex, currentPullRequestIndex)`.

Reply:
Well… yeah… I had to guess. :upside_down_face: